doc/man/salt.7

```diff
@@ -110489,6 +110489,12 @@ A module for shelling out.
 T}
 _
 T{
+\fBcohesity\fP
```

Comment:
Don't change manpages. That's done by us right before a release

Reply:
Okay